content/blog/2023/sqlalchemy-for-django-developers.md

```diff
@@ -192,14 +205,14 @@ achieves this by exposing custom methods as part of the class attributes:
 users = session.scalars(select(User).where(User.id.in_([1, 2, 3]))).all()
 ```
 
-*The trailing underscore in `in_()` is needed because `in` is a reserved word in
-Python, not because of anything specific to SQLAlchemy.*
+<!-- prettier-ignore -->
```

Comment:
I'm curious why this was needed -- we don't run Prettier on Markdown files by default. Is this a setting in your local IDE?

Reply:
Ah, that must be it. And it look like it's adding that blank line before the closing `{% endcallout %}` as well. I'll disable it